metadata_backend/api/handlers/object.py

```diff
@@ -161,6 +163,10 @@ async def post_object(self, req: Request) -> Response:
         patch = await self.prepare_folder_patch_new_object(collection, ids, patch_params)
         await folder_op.update_folder(folder_id, patch)
 
+        # Create draft dataset to Metax catalog
+        if collection in {"study", "dataset"}:
```

Comment:
maybe we should put a restriction for `/drafts` as we don't need to create metax entries for those

Reply:
it is o like this, my bad.